catkin_ws/src/vision/src/point_cloud.py

```diff
@@ -78,6 +63,27 @@ def convert_from_uvd(color, z_map):
         header.frame_id = "auv_base"
         pub_msg = point_cloud2.create_cloud(header=header, fields=fields, points=xyz_rbg_img)
         return pub_msg
+
+def get_point_cloud_image():
+    if y_over_z_map is not None:
+        xyz_rbg_img = get_xyz_image(rgb, depth)
+        point_cloud_img = bridge.cv2_to_imgmsg(np.float32(xyz_rbg_img[:,:,:3]), "bgr8")
+        return point_cloud_img
+
+def get_xyz_image(color, depth):
```

Comment:
since depth is a global variable no need for it to be passed as an argument I think